mlflow/cli.py

```diff
@@ -554,6 +562,16 @@ def gc(older_than, backend_store_uri, run_ids):
     else:
         run_ids = run_ids.split(",")
 
+    mlflow_client = mlflow.MlflowClient()
+    deleted_experiments = [
+        experiment
+        for experiment in mlflow_client.search_experiments(view_type=ViewType.DELETED_ONLY)
```

Comment:
`MlflowClient.search_experiments` is paginated and doesn't fetch all experiments. Can we use `mlflow.search_experiments`?

Reply:
For `.search_runs`, which one is recommended `MlflowClient.search_runs` or `mlflow.search_runs # this is returning DataFrame`?

Reply:
`mlflow.search_runs`. `MlflowClient.search_runs` is also paginated.